app/pages/shows/_slug/_id.vue

```diff
@@ -185,8 +214,7 @@ export default {
       return this.$moment(this.arcsiEpisode.play_date).format('yyyy. MMMM Do.')
     },
     episodeImage () {
-      const rootLink = mediaServerURL + this.slug + '/'
-      return rootLink + this.arcsiEpisode?.image_url
+      return this.arcsiEpisode?.image_url
```

Comment:
Oh I see. Thanks for pointing it out! But does it actually cause any trouble? I'm asking because the images seem to be displayed correctly on show subpages.

Reply:
It doesn't make any trouble, works as it should, just looks a bit ugly for my eyes 😅